composer.lock

```diff
@@ -4,15 +4,16 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "cabdf41576f13b9aeb418d860129cd9d",
+    "content-hash": "167f5c90d0f8c7224fdc8707e1ba107f",
     "packages": [
         {
             "name": "automattic/jetpack-logo",
-            "version": "1.0.0",
+            "version": "dev-update/dont-symlink-packages",
```

Comment:
@oskosk we'll need to build a new `composer.lock` when in the `master` branch to avoid this reference.